instrumentation/net/http/otelhttp/handler.go

```diff
@@ -201,12 +205,15 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	labeler := &Labeler{}
 	ctx = injectLabeler(ctx, labeler)
 
+	// Add metrics
+	attributes := append(labeler.Get(), semconv.HTTPServerMetricAttributesFromHTTPRequest(h.operation, r)...)
```

Comment:
This breaks the use of `Labeler` to allow handlers to add labels to the instrumentation.